### PR TITLE
Client Skeleton

### PR DESCRIPTION
**Pull request by Denys-Bykov-0**

### CODE REVIEW

File: src/Client/CMakeLists.txt [hunk 1]
```diff
@@ -23,23 +27,49 @@ list(APPEND Spend-Note-Client_LIBS
...
 target_link_libraries(
     Spend-Note-Client
     PUBLIC
     ${Spend-Note-Client_LIBS}
 )
+
+target_link_libraries(Spend-Note-Client PRIVATE )
+
+set_target_properties(Spend-Note-Client PROPERTIES
+    MACOSX_BUNDLE_GUI_IDENTIFIER my.example.com
```
Comment (iodream) [inline src/Client/CMakeLists.txt:68]: is this a leftover or meant to be example?

Comment (Denys-Bykov-0) [reply src/Client/CMakeLists.txt:68]: I think that just irrelevant peace of code automatically generated by QT.

**Review by VasylDobrovolskyi — APPROVED** [review]
Seems good

**Review by MaksymTerekhovskyy — APPROVED** [review]

**Review by iodream — APPROVED** [review]

**Review by sorbeee — COMMENTED** [review]
Seems good

**Pull request merged**